browser/ui/views/sidebar/sidebar_container_view.h

```diff
@@ -100,6 +112,8 @@ class SidebarContainerView
   base::ScopedObservation<sidebar::SidebarModel,
                           sidebar::SidebarModel::Observer>
       observed_{this};
+  std::unique_ptr<views::MenuRunner> context_menu_runner_;
+  std::unique_ptr<ui::MenuModel> context_menu_model_;
```

Comment:
`context_menu_model_` should go before `context_menu_runner_`. `runner` might interact with `model` on destruction and looks like it's actually happening, this is what I found:
```
MenuModelAdapter::~MenuModelAdapter() {
  if (menu_model_)
    menu_model_->SetMenuModelDelegate(nullptr);
}
```